javascript/javascript1/week1/dogyearcalculator.js

```diff
@@ -0,0 +1,9 @@
+let dogYearOfBirth = 2015;
```

Comment:
Don't use 'let' to declare the variable unless you want to reuse the variable within the code.
Always use 'const' for declarations.